core/App/components/buttons/Button.tsx

```diff
@@ -13,13 +13,14 @@ export interface ButtonProps {
   title: string
   buttonType: ButtonType
   accessibilityLabel?: string
+  icon?: Element
   testID?: string
   onPress?: () => void
   disabled?: boolean
 }
 
 export const Button: React.FC<ButtonProps & React.RefAttributes<HTMLInputElement | undefined>> = forwardRef(
-  ({ title, buttonType, accessibilityLabel, testID, onPress, disabled = false }, ref: any) => {
+  ({ title, buttonType, accessibilityLabel, icon, testID, onPress, disabled = false }, ref: any) => {
```

Comment:
Could we just allow consumers to pass in a child element rather than pass an icon by props? 

@jleach thoughts?

Reply:
Like `<Button><Icon /></Button>` I think that would be preferred but can we make it look like the icon is part of the button that way?

Reply:
Got it, I'll make the change